services/api/service.go

```diff
@@ -1267,9 +1467,14 @@ func (api *RelayAPI) handleBuilderGetValidators(w http.ResponseWriter, req *http
 	}
 }
 
-func (api *RelayAPI) handleSubmitNewBlock(w http.ResponseWriter, req *http.Request) {
-	receivedAt := time.Now().UTC()
+func (api *RelayAPI) handleSubmitNewBlock(w http.ResponseWriter, req *http.Request) { //nolint
```

Comment:
`nolint` is too broad! please specify what exactly you don't want to lint (i.e. `nolint:gocognit`)

Reply:
done!